src/ResourceManager/AzureBatch/Commands.Batch/Models.Generated/PSCloudPool.cs

```diff
@@ -233,11 +272,19 @@ public PSCloudServiceConfiguration CloudServiceConfiguration
             }
         }
         
-        public System.Int32? CurrentDedicated
+        public System.Int32? CurrentDedicatedComputeNodes
```

Comment:
Add mapping to CurrentDedicated and obsolete it.

Reply:
My intent is to just leave this as is (no obsolete) since this is a breaking change.

Reply:
As long as this is documented in the upcoming breaking changes document, this is ok.

Reply:
Added docs